src/App.js

```diff
@@ -65,15 +93,15 @@ function App() {
                 value={password}
                 ref={textInput}
               />
-              <div className="copied">{copied ? 'copied' : ''}</div>
+              <div>{pronunciation}</div>
```

Comment:
also I think, we should put some notes in UI, what is this component/ui meant to be?

Reply:
what if is not `kata` then shouldn't show this pronunciation @imballinst ?

Reply:
Ah, yes, something like "Cara pelafalan:", perhaps? How about that?

I think if it's random letter, this "feature" also supports the original idea of this app, right? To make randomized password easier to pronounce?

Reply:
okay that's true also, sure no need to flag.

`Pelafalan` is okay